kolibri/core/assets/test/heartbeat.spec.js

```diff
@@ -0,0 +1,221 @@
+/* eslint-env mocha */
```

Comment:
should probably change this to /* eslint-env jest */

Reply:
Yeah, can actually get rid of all of these, because I just added the jest env to the base eslint config.

Reply:
And will set that to only apply to .spec.js files, so that we don't let jest globals weirdly sneak into our client source code.